includes/polling/applications/php-fpm.inc.php

```diff
@@ -4,7 +4,7 @@
 
 $options      = '-O qv';
 $mib          = 'NET-SNMP-EXTEND-MIB';
-$oid          = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.8.112.104.112.102.112.109.115.112';
+$oid          = 'NET-SNMP-EXTEND-MIB::nsExtendOutputFull.8.112.104.112.102.112.109.115.112';
```

Comment:
Thanks but you don't or shouldn't need NET-SNMP-EXTEND-MIB:: as you've already specified the mib above.